### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by malwini**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -25,16 +57,14 @@ Assume she buys 12.3 megawatt within the hour 8 to 9 for 20.5 euros/megawatt-hou
 }
 ```
 
-It is essential that we maintain an open connection to the exchange at all times so that we don't miss any deals).
+Trades are the most important piece of data for our operation, they are the only way for us to keep track of the quantities we are trading. We can receive up to a 100 trades per minute at the moment.
```
Comment (mxauriga) [inline Readme.md:60]: nice!

File: Readme.md [hunk 2]
```diff
@@ -86,33 +116,41 @@ paths:
...
+There is a limit on the number of requests we can submit to the API, about 50 per minute and 1000 per hour.
 
 ## Reporting:
 
 The reporting frontend displays the following dashboard:
 ![img.png](dashboard.png)
-It can be accessed by the traders from their browsers.
-The dashboard is updated with all the current values every 5 Minutes. 
+
+It can be accessed by the traders from their browsers. Since this is the traders' main source of validation, they might get impatient around the time the reference price is supposed to be published and start massively refreshing their reporting dashboard.
```
Comment (mxauriga) [inline Readme.md:131]: Maybe this is a little bit too close to the truth. 😆  Maybe this frighten candidates. 😅

File: Readme.md [hunk 3]
```diff
@@ -86,33 +116,41 @@ paths:
...
 - which components would you need? How do you decide on the boundaries between these components?
-- which databases or any other data management systems?
+- which databases or any other data management systems would you need?
 - how would these components communicate with each other?
 - which technologies would you use for each component and why?
-- How and where would you run such a system? Which cloud services would you use?
-- How would you management updates and deployments in such a system?
-- Anything else that you think is relevant
+- How and where would you run such a system in the cloud? Which services would you use?
+- How would you manageme updates and deployments in such a system?
```
Comment (mxauriga) [inline Readme.md]:
```suggestion
- How would you manage updates and deployments in such a system?
```

File: Readme.md [hunk 4]
```diff
@@ -86,33 +116,41 @@ paths:
...
+- How would you structure logging and monitoring?
```
Comment (mxauriga) [inline Readme.md:150]: Maybe to much? The other points are much more Important. We want candidates to focus on those, dont we?

**Review by mxauriga — APPROVED** [review]

**Comment by mxauriga** [comment]
I really like the character of the Challenge! It tells the candidate a lot about us and the work and topics he could expect! 😍

File: Readme.md (hunk 2 above)
Comment (malwini) [reply Readme.md:131]: I wanted to have something about how the users use the system and this refers to a couple of important topics like elasticity VS scalability and managing costs in the cloud.

I would like to try it and see what happens!

File: Readme.md (hunk 4 above)
Comment (malwini) [reply Readme.md:150]: These are all more like inspirations than actual questions, there is no time to tackle all of these anyway. I think each candidate would focus on some specific aspects, maybe what they know best?

**Pull request merged**